.github/workflows/service_test_hdfs.yml

```diff
@@ -173,3 +173,41 @@ jobs:
           OPENDAL_HDFS_ATOMIC_WRITE_DIR: /tmp/atomic_write_dir/opendal/
           OPENDAL_HDFS_NAME_NODE: default
           OPENDAL_HDFS_ENABLE_APPEND: false
+
+  hdfs-default-gcs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Rust toolchain
+        uses: ./.github/actions/setup
+        with:
+          need-nextest: true
+
+      - name: Setup java env
+        uses: actions/setup-java@v4
+        with:
+          distribution: temurin
+          java-version: "11"
+      - name: Setup hadoop env
+        shell: bash
+        run: |
+          curl -LsSf https://dlcdn.apache.org/hadoop/common/hadoop-3.3.5/hadoop-3.3.5.tar.gz | tar zxf - -C /home/runner
+          curl -LsSf -o /workspaces/opendal/tmp/hadoop-3.3.5/share/hadoop/common/lib/gcs-connector-hadoop2-2.2.19-shaded.jar  https://github.com/GoogleCloudDataproc/hadoop-connectors/releases/download/v2.2.19/gcs-connector-hadoop2-2.2.19-shaded.jar
```

Comment:
Is it ok to download into `$HADOOP_HOME/share/hadoop/common/lib`?

Reply:
sure. this was a mistake. updated.